RockLib.Configuration.ObjectFactory/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+#### Changed
+
+- Updated `ToCompare()` method in `ConstructorOrderInfo` to also compare the parameter types during comparison.
```

Comment:
Oops typo in method name.  Should be CompareTo()

Reply:
Could be fixed in the release commit though.

Reply:
Good point, but too late lol.